docs/modules/ROOT/pages/contributor-guide/create-new-extension.adoc

```diff
@@ -36,10 +36,12 @@ The above sequence of commands does the following:
 * It creates a basic `FooAbcProcessor` class in the deployment module.
 * It also creates a stub of an integration test module under `integration-tests/foo-abc`.
 +
+6. Add the extension name in the rigth category in  `tooling/scripts/test-categories.yaml` file.
```

Comment:
```suggestion
6. Add the extension name to the right category in  `tooling/scripts/test-categories.yaml` file.
```

Reply:
@jamesnetherton thanks. I made the changes